benchmarks/01_trivial/08_masked_load_store.ispc

```diff
@@ -0,0 +1,19 @@
+#define MASKED_LOAD_STORE(T_ISPC)                                                                                      \
+    export void masked_load_store_##T_ISPC(uniform T_ISPC aIN[], uniform T_ISPC bIN[], uniform T_ISPC cOUT[],          \
+                                           uniform int n, uniform int aMask[]) {                                       \
```

Comment:
I would use `bool` for the mask, hope this work without issues in the C++ header.